yellow-paper/docs/decentralization/actors.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Nodes and Actors
+---
+
+To analyze the suitability of different node types for various actor types in the system, let's break down the node types and actor types, considering the characteristics, preferences, abilities, and potential motivations of each actor. This will help determine which actors are likely to run which nodes and how they might interact with the Private eXecution Environment (PXE).
+
+## Node Types
+
+The below node types are ordered in resource requirements. The types exists both in the context of Aztec but also other blockchains. All the nodes, participate in the peer-to-peer (p2p) network, but with varying capacity.
+
+1. **Light Node**: 
+    - Follows the blockchain but only interested in headers (stores headers).
+    - Minimal resource requirements.
```

Comment:
Perhaps we need to link to Zac's doc in the ../cryptography/ section, which discusses target resource requirements.

Reply:
Whoops, I forgot to "start a review". Sorry for the spam.